Modules/System/BLOB Storage/src/TempBlobListImpl.Codeunit.al

```diff
@@ -44,13 +44,20 @@ codeunit 4109 "Temp Blob List Impl."
     end;
 
     procedure RemoveAt(Index: Integer): Boolean
+    begin
+        exit(RemoveRange(Index, 1));
+    end;
+
+    procedure RemoveRange(Index: Integer; ElementCount: Integer): Boolean
     var
         TempBlobList: Codeunit "Temp Blob List";
     begin
         if not TempBlobRec.Get(Index) then
             Error(ObjectDoesNotExistErr, Index);
-        if TempBlobRec.Get(Index + 1) then
-            GetRange(Index + 1, Count() - Index, TempBlobList);
+
+        if TempBlobRec.Get(Index + ElementCount) then
+            GetRange(Index + ElementCount, Count() - Index - ElementCount, TempBlobList);
```

Comment:
This PR about is parametrizing the `1`. 

However, `Count() - Index` became `Count() - Index - ElementCount`. Why is that? Was this an issue before?

Reply:
That's a bug, indeed. Existing tests did not cover this verification. Fixed and added tests.